test/NuGet.Clients.Tests/NuGet.PackageManagement.VisualStudio.Test/NuGet.PackageManagement.VisualStudio.Test.csproj

```diff
@@ -27,6 +27,7 @@
     <Compile Include="Feeds\SourceRepositoryCreator.cs" />
     <Compile Include="Feeds\SourceRepositoryExtensionsTests.cs" />
     <Compile Include="CpsPackageReferenceProjectTests.cs" />
+    <Compile Include="GlobalSuppressions1.cs" />
```

Comment:
Probably shouldn't be GlobalSuppressions1.

Reply:
I agree the reason why `GlobalSuppressionsX` files are created is because we have a shared suppression file here https://github.com/NuGet/NuGet.Client/blob/dev/build/TestShared/GlobalSuppressions.cs. Almost all the test projects has reference to the the shared suppressions file

Reply:
![image](https://user-images.githubusercontent.com/52756182/135521879-740f859d-3058-4b60-9588-63d443483d3c.png)

The yellow marked file is pointing to the shared file.